src/upgrades/mod.rs

```diff
@@ -1134,6 +1165,26 @@ mod tests {
         );
     }
 
+    #[test]
+    fn prepare_then_upgrade_0_14_2() {
+        test_upgrade(
+            "test-resources/migrations/v0_14_2/",
+            &[
+                "ca_objects",
+                "cas", // testbed/command-6.json was intentionally removed
```

Comment:
"Removed" - is that right? I don't see it in earlier test-resources/migrations/xxx/cas/ directories so it wasn't removed compared to them, but it is _missing_ in the sequence as there is command-5.json and command-7.json but no command-6.json.